docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: ./.config
       args:
-        grafana_version: ${GRAFANA_VERSION:-9.3.8}
+        grafana_version: ${GRAFANA_VERSION:10.4.0}
```

Comment:
Is this tested? Previously this string had a hyphen in it.

Reply:
This failed when attempting `make compose`, but works when the missing hyphen is added back in.